### PR TITLE
- Provide initial content for the README.md file

### PR DESCRIPTION
**Pull request by rjschwei**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
...
+
+The SUSE-Enceladus organization in GitHub was created to serve the needs of the SUSE Public Cloud DevOps team. We started out with a "collector" project named Enceladus within the SUSE organization. Over time the collector project grew unwieldy and we had various unrelated source bases intermingled in the Enceladus project. In addition some of our other projects, large an small were maintained in separate projects in separate orgs. SUSE-Enceladus was created to collect our efforts for original source code in one organization, help us be more organized, and provide a better overview of the projects we develop and maintain. The initial projects in this organization started as migrations from Enceladus or other projects we maintained in GitHub.
+
+## Goals
+
+We strive to provide a reasonably uniform experience for all projects within the SUSE-Enceladus organization. In an effort to achieve this uniform experience we strive to document our processes and procedures. Each project will have a LICENSE, a README.md, and a CONTRIBUTING.md file that provides information about he project. However, projects do have different needs and therefore some differences are to be expected.
```
Comment (smarlowucf) [inline README.md]: s/he/the

File: README.md [hunk 2]
```diff
@@ -1,2 +1,14 @@
...
+
+Projects in the SUSE-Enceladus organization represent original work of the SUSE Public Cloud DevOps team. We hope our work is useful to you and we welcome any contribution. The project is in alignment with the SUSE open source policy [1] and as such we do not have a contributor agreement.
+
+[1] https://opensource.suse.com/suse-open-source-policy
```
Comment (smarlowucf) [inline README.md]: Can use markdown link instead.

**Review by smarlowucf — COMMENTED** [review]
Good. We probably want this visible in the repo and in the static page. I think we can drop the index.md and jekyll will look for a README.md to use as the index. But we will need to update the _config to not ignore README.md.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,14 @@
...
+
+## Goals
+
+We strive to provide a reasonably uniform experience for all projects within the SUSE-Enceladus organization. In an effort to achieve this uniform experience we strive to document our processes and procedures. Each project will have a LICENSE, a README.md, and a CONTRIBUTING.md file that provides information about he project. However, projects do have different needs and therefore some differences are to be expected.
+
+Projects in the SUSE-Enceladus organization represent original work of the SUSE Public Cloud DevOps team. We hope our work is useful to you and we welcome any contribution. The project is in alignment with the SUSE open source policy [1] and as such we do not have a contributor agreement.
```
Comment (schaefi) [inline README.md]: "The project is in alignment..." On the first read I thought "which project", so this covers all projects in the SUSE-Enceladus organisation correct ?

Comment (rjschwei) [reply README.md]: > "The project is in alignment..." On the first read I thought "which project", so this covers all projects in the SUSE-Enceladus organisation correct ?

Yes, covers all projects in SUSE-Enceladus, clarified.

**Comment by rjschwei** [comment]
I'll leave the fiddling with the jekyll files to you, @smarlowucf , as I know nothing about how any of these pieces fit together, and I have little desire to figure it out.

**Comment by smarlowucf** [comment]
> I'll leave the fiddling with the jekyll files to you, @smarlowucf

Okay, or if @bear454 wants to do some fiddling with this? We can simplify things if we stick to one of the themes that's integrated into GitHub as well.

**Comment by smarlowucf** [comment]
In regards to the link it could be inline:

```
[SUSE open source policy](https://opensource.suse.com/suse-open-source-policy)
```

Or reference style:

```
[SUSE open source policy][1]

 [1]: https://opensource.suse.com/suse-open-source-policy
```

**Review by smarlowucf — APPROVED** [review]

**Review by bear454 — APPROVED** [review]

**Pull request merged**